US-CBP/cbp-theme
# Merge Universal Header & Application Header

#### What's this PR do?
* universal header and application header merged under `universalHeader.html`
* css for both headers merged under `cbp-header.scss`
* small responsive fixes

#### Where should the reviewer start?
* `universalHeader.html`

#### How should this be manually tested?
* open `universalHeader.html` and check to see that both components are functioning normally under different responsive points.

#### Any background context you want to provide?
* phones that have a width of 300px or below will have clipping issues.
* phones with a width of 375px or below will have the cbp logo text hidden to accommodate the buttons.

#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Questions:
- Is there a blog post?
- Does the knowledge base need an update?
